integration-test-scripts/verdaccio-config.yaml

```diff
@@ -0,0 +1,35 @@
+# Modular Verdaccio Config for CI
+
+storage: ./storage
+plugins: ./plugins
+
+web:
+  title: Verdaccio
+
+auth:
+  htpasswd:
+    file: ./htpasswd
+
+# Back Verdaccio onto Artifactory instead of NPM
```

Comment:
This comment should be removed (no usage of artifactory here)

Reply:
Good spot, thanks